lms/djangoapps/course_api/blocks/permissions.py

```diff
@@ -6,7 +6,15 @@
 from student.models import CourseEnrollment
 
 
-def can_access_other_users_blocks(requesting_user, course_key):
+def can_access_all_blocks(requesting_user, course_key):
+    """
+    Returns whether the requesting_user can access all the blocks
+    in the course.
+    """
+    return has_access(requesting_user, 'staff', course_key)
```

Comment:
Ok!  It's up to you--just thought it would ensure that the role naming were ever changed (however unlikely), it would be caught here.